docker-compose.yaml

```diff
@@ -7,11 +7,12 @@ services:
     command: --max_allowed_packet=32505856
     restart: always
     volumes:
-      - ./tessEBs.20250323.sql:/docker-entrypoint-initdb.d/dbimport.sql
-      - ./dbdata:/var/lib/mysql
+      - ./tessEBs.20250323.sql:/docker-entrypoint-initdb.d/dbimport.sql:ro
+      - dbdata:/var/lib/mysql
+      - ./my.cnf:/etc/mysql/conf.d/my.cnf:ro
       - /tmp/tessEBs/mysqld:/var/run/mysqld
     healthcheck:
-      test: ["CMD", "mariadb", "-h", "localhost", "-u${MYSQL_USER}", "-p${MYSQL_PASSWORD}", "-e", "use", "${MYSQL_DATABASE}"]
+      test: ["CMD", "mariadb", "-h", "127.0.0.1", "-u${MYSQL_USER}", "-p${MYSQL_PASSWORD}", "-e", "use", "${MYSQL_DATABASE}"]
```

Comment:
this might not be generally compatible, so perhaps it's best we leave it as localhost, and you keep your local stash if localhost still doesn't work on your end